doc/classes/CanvasItem.xml

```diff
@@ -569,13 +569,14 @@
 			The rendering layers in which this [CanvasItem] responds to [Light2D] nodes.
 		</member>
 		<member name="material" type="Material" setter="set_material" getter="get_material">
-			The material applied to textures on this [CanvasItem].
+			The material applied to this [CanvasItem].
 		</member>
 		<member name="modulate" type="Color" setter="set_modulate" getter="get_modulate" default="Color(1, 1, 1, 1)">
-			The color applied to textures on this [CanvasItem].
+			The color applied to this [CanvasItem].
 		</member>
 		<member name="self_modulate" type="Color" setter="set_self_modulate" getter="get_self_modulate" default="Color(1, 1, 1, 1)">
-			The color applied to textures on this [CanvasItem]. This is not inherited by children [CanvasItem]s.
+			The color applied to this [CanvasItem]. Does not propagate to children [CanvasItem]s.
+			[b]Note:[/b] Internal children (e.g. sliders in [ColorPicker] or tab bar in [TabContainer]) are also not affected by this property (see [code]internal[/code] parameter in [method Node.add_child]).
```

Comment:
Not sure it's worth pointing to the internal parameter of `add_child`, as this is mostly relevant with built-in nodes and not complex widgets created by users. Could perhaps point to get/find methods, and their `internal` flag, to give a clue about how to work around this limitation?

Reply:
Made it like that because `add_child` has the most extensive explanation of what internal children are, other Node properties refer to it like that too. But indeed pointing to `(get|find)_child` (one of them? both? :thinking: ) probably makes more sense as it gives a hint what can be used to obtain them. If the user don't know what the internal children are though then they would be redirected further to `add_child`'s description from there but that's likely fine. :upside_down_face:

Reply:
Can also add an explanation to get/find methods, what internal nodes are. Duplication is not bad per se, as sometimes descriptions are displayed standalone and not in a context of the entire documentation (e.g. in tooltips).

Reply:
> Can also add an explanation to get/find methods, what internal nodes are.

I think I'll pass on this one. If the user already got from the description of `modulate`/`self_modulate` to the description of `(get|find)_child` then it shouldn't be a problem to go one link deeper (they must have already been able to follow a link anyway).